src/hooks/useChainId.ts

```diff
@@ -38,13 +38,7 @@ export const useUrlChainId = (): string | undefined => {
   const { prefix } = parsePrefixedAddress(safe)
   const shortName = prefix || chain
 
-  if (shortName) {
-    const chainId = Object.entries(chains).find(([key]) => key === shortName)?.[1]
-    if (chainId == null) {
-      throw Error('Invalid chain short name in the URL')
-    }
-    return chainId
-  }
+  return Object.entries(chains).find(([key]) => key === shortName)?.[1]
```

Comment:
By removing the error we now need to check that the returned value exists where it's used. We could fallback to an empty string but I think that could cause issues down the line.

I'm surprised TypeScript didn't pick up on this.

Reply:
Let's return the default chainId as a fallback.

Reply:
This returns undefined in case it can't find the chain in our config. The behaviour should be the same as what we had previously. The fallbacks to the last session chain id and default chain id are already in place in `useChainId`.